tests/drivers/fprotect/negative/src/main.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/* This test checks if fprotect have been set up correctly and is able to
+ * protect the flash from being modified. We should be able to read the flash
+ * but not modify it. This test should fail with an BUS fault when trying to
+ * write to an address below the location of b0(Secure bootloader) as it should
+ * have been set up to be protected by SPU/ACL etc. depending on the chip used
+ */
+
+#include <ztest.h>
+#include <device.h>
+#include <flash.h>
+
+u32_t valid_write_addr = 0x1C000;
+
+
+void test_flash_write_protected(void)
```

Comment:
This should be `static`, `valid_write_addr` as well.

Reply:
Done